### ISSUE TITLE
Add cold start AUC evaluation

### ISSUE THREAD

**Issue by aaossa** [aaossa/VBPR-PyTorch#12]
From the paper,

> For each dataset, we report the average AUC on the full test set T (denoted by ‘All Items’), as well as **a subset of T which only consists of items that had fewer than five positive feedback instances in the training set (i.e., cold start)**. These cold start items account for around 60% of the test set for the two Amazon datasets, and 80% for Tradesy.com; this means for such sparse real-world datasets, a model must address the inherent cold start nature of the problem and recommend items accurately in order to achieve acceptable performance.
